packages/react-server-cli/README.md

```diff
@@ -115,6 +115,12 @@ Sets the severity level for the logs being reported. Values are, in ascending or
 
 Default is **'debug'** in development mode and **'notice'** in production.
 
+#### --global-react
+
+Expose `React` as a global variable so modules that use JSX don't have to import it (without obviously using it).
+
+Defaults to **false**.
+
```

Comment:
It's all right, though not my 100% favorite, for a few reasons:

- Just a general aversion to magic globals when a module system exists. Also, does this put a React global in both server and client global scope?
- Components that are written assuming a global React will not work in environments that use standard module import, which then forces more apps to add React to global scope if they want to share that code. In this way, it infects the ecosystem and causes compatibility problems.
- It maybe has the potential to confuse packagers and other new dev tools that depend on ES2015 imports to understand module dependency graphs.
- If the issue is that you don't want to have to type `import React from 'react'` at the top of component files with JSX and no reference to React, there is [a simple babel transform](https://www.npmjs.com/package/babel-plugin-react-require) that will do that for you. The output of your babel build will then be compatible with projects that don't use global React and will have parseable dependency trees.

All that said, would I really mind terribly? No, not really. Your call!